### ISSUE TITLE
Add a CNAME for the helm chart URL

### ISSUE THREAD

**Issue by arschles** [gomods/athens#1101]
**Is your feature request related to a problem? Please describe.**

Right now, the helm charts are hosted at an Azure blob storage URL (see https://github.com/gomods/athens/pull/1097#issuecomment-468836384). It would be nice to have that behind a cleaner URL

**Describe the solution you'd like**

Maybe have `charts.gomods.io` be the place you `helm install` from?

**Describe alternatives you've considered**

We could leave it as-is

**Additional context**

This is a follow-up to #1097

**Comment by chriscoffee** [comment]
I think `charts.gomods.io` too! This is blocked by #1100 right?

**Comment by arschles** [comment]
@chriscoffee yup, it is. You reminded me to get #1100 done!

**Comment by DrPsychick** [comment]
Does this qualify as "cleaner URL" ? https://gomods.github.io/athens-charts

Then no CNAME is required, but the domain could be used for "marketing" purposes. But in any case, setting up a CNAME and adding it to GitHub pages can be done within minutes.

**Comment by arschles** [comment]
@DrPsychick I think that qualifies, yes! thanks
